addons/product/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     'name': 'Products & Pricelists',
     'version': '1.2',
     'category': 'Sales/Sales',
-    'depends': ['base', 'mail', 'uom'],
+    'depends': ['base', 'mail', 'uom', 'base_printer'],
```

Comment:
```suggestion
    'depends': ['base', 'mail', 'uom'],
```

auto install based on `mail`